php/libraries/ProjectDefault.class.inc

```diff
@@ -85,7 +85,7 @@ class ProjectDefault extends Project
     }
 
     /**
-     * Get that project subprojects as defined in the project_rel table
+     * Get that project subprojects as defined in the project_subproject_rel table
```

Comment:
This is not accurate. Project default gets all subprojects because it is meant to be used when there is no project

Reply:
Otherwise, it seems fine

Reply:
fixed